README.md

```diff
@@ -85,3 +85,11 @@ To set up your local environment to run the unit tests:
     Time: 703 ms, Memory: 33.75Mb
     OK (1 test, 3 assertions)
     ```
+
+To set up PHPCodesniffer to test changes for adherance to WordPress Coding Standards
+
+1. install [PHPCS](https://github.com/squizlabs/PHP_CodeSniffer).
```

Comment:
minor point: in the future, you'll want to pay attention to details like capitalization (here and below), punctuation (period below), ```s around command lines, etc.

definitely not a big deal. we can clean these up later, any time.